pkgs/by-name/pr/protoc-gen-lint/package.nix

```
@@ -0,0 +1,26 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+
+buildGoModule (finalAttrs: {
+  pname = "protoc-gen-lint";
+  version = "0.3.0";
+
+  src = fetchFromGitHub {
+    owner = "ckaznocha";
+    repo = "protoc-gen-lint";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-8+fPkXmigP8ZqcFGCnw1KZhJQcahDjKnZUJ1eqaHhs0=";
+  };
+
+  vendorHash = null;
+
+  meta = with lib; {
+    description = "A plug-in for Google's Protocol Buffers compiler to check .proto files for style violations";
+    homepage = "https://github.com/ckaznocha/protoc-gen-lint";
+    license = licenses.mit;
+    maintainers = with lib.maintainers; [ jherland ];
+  };
```

```suggestion
  meta = {
    description = "Plug-in for Google's Protocol Buffers compiler to check .proto files for style violations";
    homepage = "https://github.com/ckaznocha/protoc-gen-lint";
    license = lib.licenses.mit;
    maintainers = with lib.maintainers; [ jherland ];
  };
```

> Description: Not start with definite (“The”) or indefinite (“A”/“An”) article.

https://nixos.org/manual/nixpkgs/stable/#var-meta-description

Plus reduction in wide usage of `with lib;`

---

If you'd like help maintaining this package feel free to add me to the maintainers list as `jk`
:slightly_smiling_face: